File: notify/sendToList2.py
# Documentation: https://www.twilio.com/notify/api
# Documentation: https://www.twilio.com/docs/notify/api/notifications
# ------------------------------------------------------------------------------
print("+++ Send notifications.")
import os
import json
from twilio.rest import Client
account_sid = os.environ.get("ACCOUNT_SID")
auth_token = os.environ.get("AUTH_TOKEN")
client = Client(account_sid, auth_token)
#
phoneNumber1 = os.environ.get("PHONE_NUMBER1")
phoneNumber2 = os.environ.get("PHONE_NUMBER2")
print("+ Send notifications to: " + phoneNumber1 + " and " + phoneNumber2 )
print("+ Using notification service, SID: " + os.environ.get("NOTIFY_SERVICE_SID"))
notify_service_sid = os.environ.get("NOTIFY_SERVICE_SID")
notification = client.notify.services(notify_service_sid).notifications.create(
    body='Hello to the list of 2.',
    to_binding=[
        json.dumps({"binding_type":"sms", "address": phoneNumber1}),
        json.dumps({"binding_type":"sms", "address": phoneNumber2})
    ]
  )
print("+ Notification SID: " + notification.sid)
print("+++ Exit.")
# ------------------------------------------------------------------------------

File: notify/sendToList2.php
<?php
require __DIR__ . '/../../twilio-php-master/Twilio/autoload.php';
use Twilio\Rest\Client;
$twilio = new Client(getenv('ACCOUNT_SID'), getenv('AUTH_TOKEN'));
$notification = $twilio->notify->v1->services(getenv('NOTIFY_SERVICE_SID'))
        ->notifications
        ->create(array(
            "body" => "hello 2.2",
            "toBinding" => array(
                "{\"binding_type\":\"sms\", \"address\":\"" . getenv('PHONE_NUMBER1') . "\"}",
                "{\"binding_type\":\"sms\", \"address\":\"" . getenv('PHONE_NUMBER4') . "\"}"
                )
        )
);
echo "+ Notification SID: " . $notification->sid . "\xA";
?>
